.github/workflows/deploy.yml

```
@@ -38,3 +38,6 @@ jobs:
                       cd devoops
                       git pull
                       ./deploy.sh
+            - name: GH Release
+              uses: softprops/action-gh-release@v0.1.5
```

According to the action you're using (https://github.com/softprops/action-gh-release) the GITHUB_TOKEN is required.
```suggestion
              uses: softprops/action-gh-release@v0.1.5
              env:
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```